src/erc721/ERC721.sol

```diff
@@ -1,14 +1,13 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity >=0.8.0;
 
-/// @notice Modern and gas efficient ERC-721 + EIP-2612-like implementation.
-// ! TO DO - add safeTransfer stuff 🙏
+/// @notice Modern and gas efficient ERC-721 + ERC-20/EIP-2612-like implementation.
 contract ERC721 {
     /*///////////////////////////////////////////////////////////////
                                   EVENTS
     //////////////////////////////////////////////////////////////*/
     
-    event Transfer(address indexed from, address indexed to, uint256 indexed tokenId);
+    event Transfer(address indexed owner, address indexed to, uint256 indexed tokenId);
```

Comment:
hmm i like `from`

Reply:
yeah, I actually mulled this for an inordinate amount of time lol. But, yea 'from' pairs better to 'to' and would match up with ERC20 .... foine foine ;0